external-crates/move/move-stdlib/src/natives/debug.rs

```diff
@@ -191,10 +191,8 @@ pub fn make_all(
 mod testing {
     use move_binary_format::errors::{PartialVMError, PartialVMResult};
     use move_core_types::{
-        account_address::AccountAddress,
-        language_storage::TypeTag,
-        value::{MoveStruct, MoveStructLayout, MoveTypeLayout, MoveValue},
-        vm_status::StatusCode,
+        account_address::AccountAddress, annotated_value as A, language_storage::TypeTag,
+        runtime_value as R, vm_status::StatusCode,
```

Comment:
Are there tests for this file?

Reply:
There are! in the sandbox test there are tests that call into the debug module.

Reply:
Can you double check I didn't delete these :p

Reply:
Just double checked -- they still exist!